src/movement.cpp

```diff
@@ -817,19 +817,19 @@ bool MoveEvent::executeStep(Creature* creature, Item* item, const Position& pos,
 	return scriptInterface->callFunction(4, true);
 }
 
-uint32_t MoveEvent::fireEquip(Player* player, Item* item, slots_t slot, bool boolean)
+uint32_t MoveEvent::fireEquip(Player* player, Item* item, slots_t slot, bool isCheck)
```

Comment:
Sorry, I can't think of anything more meaningful than isCheck to a boolean that is true when the function is called only to check the return value :(